<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bus</class>
 <widget class="QWidget" name="Bus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bus1">
      <property name="text">
       <string>路线1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bus2">
      <property name="text">
       <string>路线2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bus3">
      <property name="text">
       <string>路线3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
